content/docs/buildpacks/language-family-buildpacks/nodejs.md

```diff
@@ -350,6 +350,22 @@ The Node.js CNB also supports simple apps that do not require third-party packag
 If no package manager is detected, the Node.js CNB will set the start command
 `node server.js`. The app name is ___not___ currently configurable.
 
+## Other Utilities
```

Comment:
Could you reframe this around the user facing value around each of these buildpacks? Example:
"Using a Procfile", "Adding Environment Variables to the Build/Run Image", "Adding Image Labels to the Build/Run Image"

Reply:
Do you want the top level `Other Utilities` title gone?

Reply:
I think that would make sense.